### DISCUSSION TITLE
Problem with conditional and log breakpoints

### DISCUSSION THREAD

**Discussion by pejax** [scalameta/nvim-metals#340] [Q&A] [answered]
I am setting a conditional breakpoint (marked 'C' in the gutter) and a log breakpoint (marked 'L' in the gutter) - the debugger does not stop at either one. The debuggee seems to hang, I have to terminate it via the DAP 'continue' command and hit 'terminate'. Regular breakpoints (marked 'B') work as expected. Below is an excerpt from the dap logfile - which look correct according to Mathias Fußenegger. Any idea what's wrong here?
Using Scala 2.13.8 under JDK 17.0.1., nvim 0.6.1, all latest nvim plugins.

Thanks
```
[ DEBUG ] 2022-02-16T19:37:22Z+0100 ] ...nvim/site/pack/packer/start/nvim-dap/lua/dap/session.lua:850 ] "request" {
	arguments = {
		breakpoints = {
			{
				line = 5,
				logMessage = "Logging..."
			},
			{
				condition = "y=9",
				line = 7
			}
		},
		lines = { 5, 7 },
		source = {
			name = "Main.scala",
			path = "/Users/peter/ramdisk/scala-debug/src/main/scala/example/Main.scala"
		}
	},
	command = "setBreakpoints",
	seq = 2,
	type = "request"
}
```

**Comment by ckipp01** [comment, marked as answer]
Just linking these together in case anyone else stumbles on this since I answered here: https://github.com/mfussenegger/nvim-dap/discussions/438#discussioncomment-2205090
